resources/views/admin/layouts/master.blade.php

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta name="description" content="Vali is a responsive and free admin theme built with Bootstrap 4, SASS and PUG.js. It's fully customizable and modular.">
+        <!-- Twitter meta-->
+        <meta property="twitter:card" content="summary_large_image">
+        <meta property="twitter:site" content="@pratikborsadiya">
+        <meta property="twitter:creator" content="@pratikborsadiya">
+        <!-- Open Graph Meta-->
+        <meta property="og:type" content="website">
+        <meta property="og:site_name" content="Vali Admin">
```

Comment:
Đoạn này có ảnh hưởng gì ko? t thấy nó toàn ảnh template ý. xong còn link online nữa